istioctl/cmd/describe.go

```diff
@@ -1186,3 +1197,87 @@ func containerReady(pod *v1.Pod, containerName string) (bool, error) {
 	}
 	return false, fmt.Errorf("no container %q in pod", containerName)
 }
+
+// describePeerAuthentication fetches all PeerAuthentication in workload and root namespace.
+// It lists the ones applied to the pod, and the current active mTLS mode.
+// When the client doesn't have access to root namespace, it will only show workload namespace Peerauthentications.
+func describePeerAuthentication(writer io.Writer, configClient istioclient.Interface, istioNamespace, workloadNamespace string, podsLabels k8s_labels.Set) error { // nolint: lll
```

Comment:
Could you also update the `tests/integration/pilot/istioctl_test.go::TestDescribe()` to cover the PeerAuthN?

Reply:
Is this test running in any job? I'm actually not sure the describe is showing DR correctly since in my env it didn't.
And will it make sense to update integration test in a separate pr?

Reply:
yes it's running in the presubmit, search for `TestDescribe` in https://storage.googleapis.com/istio-prow/pr-logs/pull/istio_istio/36062/integ-pilot-k8s-tests_istio/1461881654225145856/build-log.txt

It's fine if you want to add it in a separate PR.